src/styles/main.scss

```diff
@@ -0,0 +1,99 @@
+@import "./utils/variables";
+
+body {

```

Comment:
Avoid direct tag styling. Better use class `body`.

Reply:
Thanks a lot for review

Reply:
@OlehMy You're welcome! ✌